dogu-actions/run-test/src/main.ts

```diff
@@ -52,7 +53,21 @@ ActionKit.run(async ({ options, logger, input, deviceHostClient, consoleActionCl
 
   let appPath = '';
   const resolvedAppVersion = appVersion || process.env.DOGU_APP_VERSION || '';
-  if (resolvedAppVersion) {
+  const resolvedAppPackageName = appPackageName || process.env.DOGU_APP_PACKAGE_NAME || '';
+
+  if (resolvedAppPackageName) {
+    const currentPlatformPackageName =
+      typeof resolvedAppPackageName === 'object'
+        ? (() => {
+            const platformAppVersion = Reflect.get(resolvedAppPackageName, DOGU_DEVICE_PLATFORM) as string | undefined;
+            if (!platformAppVersion) {
+              throw new Error(`Invalid app version: ${stringify(resolvedAppPackageName)} for platform: ${DOGU_DEVICE_PLATFORM}`);
```

Comment:
Is the error message incorrect?. I think it should be "Invalid app package name: ..."